python/astro_metadata_translator/tests.py

```diff
@@ -149,8 +150,20 @@ def assertObservationInfoFromYaml(self, file, dir=None, check_wcs=True,  # noqa:
             inconsistent.
         """
         header = read_test_file(file, dir=dir)
-        self.assertObservationInfo(header, filename=file, check_wcs=check_wcs,
-                                   wcs_params=wcs_params, **kwargs)
+
+        astropy_header = Header()
```

Comment:
If I wanted to be picky it would be good if there was a comment here explaining why we are doing this astropy test and why we are using append...

Reply:
Added. Waiting for Jenkins #34240 for the earlier code changes.